ownCloudSDK/Core/Sharing/OCCore+Sharing.m

```diff
@@ -404,22 +405,22 @@ - (nullable NSProgress *)makeDecisionOnShare:(OCShare *)share accept:(BOOL)accep
 				// - files
 				[[OCShareRole alloc] initWithType:OCShareRoleTypeEditor
 						       shareTypes:OCShareTypesMaskUserShare|OCShareTypesMaskGroupShare
-						     permissions:OCSharePermissionsMaskRead|OCSharePermissionsMaskUpdate|OCSharePermissionsMaskShare
+						     permissions:OCSharePermissionsMaskRead|OCSharePermissionsMaskUpdate|(resharingSupported ? OCSharePermissionsMaskShare : 0)
 					 customizablePermissions:OCSharePermissionsMaskNone
 						       locations:OCLocationTypeFile
 						      symbolName:@"pencil"
 						   localizedName:OCLocalized(@"Editor")
-					    localizedDescription:OCLocalized(@"Edit, download, preview and share")],
+					    localizedDescription:OCLocalized(resharingSupported ? @"Edit, download, preview and share" : @"Edit, download and preview")],
 
 				// - folders
 				[[OCShareRole alloc] initWithType:OCShareRoleTypeEditor
 						       shareTypes:OCShareTypesMaskUserShare|OCShareTypesMaskGroupShare
-						     permissions:OCSharePermissionsMaskRead|OCSharePermissionsMaskUpdate|OCSharePermissionsMaskCreate|OCSharePermissionsMaskDelete|OCSharePermissionsMaskShare
+						     permissions:OCSharePermissionsMaskRead|OCSharePermissionsMaskUpdate|OCSharePermissionsMaskCreate|OCSharePermissionsMaskDelete|(resharingSupported ? OCSharePermissionsMaskShare : 0)
 					 customizablePermissions:OCSharePermissionsMaskNone
 						       locations:OCLocationTypeFolder
 						      symbolName:@"pencil"
 						   localizedName:OCLocalized(@"Editor")
-					    localizedDescription:OCLocalized(@"Upload, edit, delete, download, preview and share")],
+					    localizedDescription:OCLocalized(resharingSupported ? @"Upload, edit, delete, download, preview and share" : @"Upload, edit, delete, download and preview")],
 
```

Comment:
`OCLocalized` needs to be changed to `OCLocalizedString` to get in sync with the new translation changes before merging.

Reply:
Fixed.